src/components/Content.js

```diff
@@ -0,0 +1,24 @@
+import { Home, About, Services, Team, Appointment, FAQ, Contact, PersDataProt, PrivacyPolicy } from "../pages/index";
+
+import { Switch, Route } from "react-router-dom";
+import { routes } from "../routing/routes";
+
+const Content = () => {
+  return (
+    <>
```

Comment:
Fragment is not needed if you return one node, like <Switch> here.  Please check other places and remove if not needed.

Reply:
Removed fragments.

Reply:
Removed fragments.